matpower/matpower-converter/src/main/java/com/powsybl/matpower/converter/MatpowerExporter.java

```diff
@@ -220,11 +221,129 @@ private static void createBuses(Network network, MatpowerModel model, Context co
         createTransformerStarBuses(network, model, context);
     }
 
+    private static boolean isEmergencyLimit(LoadingLimits.TemporaryLimit limit) {
+        return limit.getAcceptableDuration() <= 60;
+    }
+
+    private static Optional<LoadingLimits.TemporaryLimit> findShortTermLimit(Stream<LoadingLimits.TemporaryLimit> limitStream) {
+        return limitStream.filter(limit -> !isEmergencyLimit(limit))
+                .max(Comparator.comparing(LoadingLimits.TemporaryLimit::getAcceptableDuration));
```

Comment:
Why a max?

Reply:
to get for instance the 20' instead of the 10'